src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbol.cs

```diff
@@ -255,6 +256,11 @@ protected void SetReturnsVoid(bool returnsVoid)
             this.flags.ReturnsVoid = returnsVoid;
         }
 
+        protected void SetIsExtensionMethod(bool isExtensionMethod)
```

Comment:
Upon further inspection, it looks like an immutability bug. Will file an issue to fix ReturnsVoid and undo this change.

---
In reply to: [123042426](https://github.com/dotnet/roslyn/pull/19703#discussion_r123042426) [](ancestors = 123042426,122884712)